Core/Inc/CAN/general_can_data_manager.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * general_can_handler.h
+ *
+ *  Created on: Oct 16, 2020
+ *      Author: renanmoreira
+ */
+
+#ifndef INC_GENERAL_CAN_HANDLER_H_
+#define INC_GENERAL_CAN_HANDLER_H_
```

Comment:
Atualizar o nome disso daqui. Quando vc cria o arquivo pelo cubeide ele ja cria esses defines certos.

```suggestion
#ifndef INC_CAN_GENERAL_CAN_DATA_MANAGER_H_
#define INC_CAN_GENERAL_CAN_DATA_MANAGER_H_
```